config/sync/editor.editor.os2loop_section.yml

```diff
@@ -0,0 +1,42 @@
+uuid: 86abbc66-e33a-4482-9ebf-129f66841866
```

Comment:
The filename should contain `os2loop_section_page`.

Reply:
Fixed